app/helpers/settings_helper.rb

```diff
@@ -2,7 +2,8 @@
 
 module SettingsHelper
   HUMAN_LOCALES = {
-    en: 'Oulipo',
+    'en-OU': 'English (Oulipo)',
```

Comment:
Want to point this out: this brings back that glyph on localization options. Swap it if you want!